AndroidAnnotations/androidannotations-core/androidannotations/src/test/java/org/androidannotations/helper/ValidatorParameterHelperTest.java

```diff
@@ -0,0 +1,321 @@
+package org.androidannotations.helper;
+
```

Comment:
I worked on my professional computer and forgot to setup eclipse formatter on it.